docs/physics/setup/plasma/helium_nlte.rst

```diff
@@ -48,4 +48,4 @@ In the TARDIS plasma, some of these equations are re-written slightly to make us
 Numerical He NLTE
 ------------------
 
-Another `helium_treatment` option offered by TARDIS is `numerical-nlte`. The use of this plasma property requires an additional code that is the property of Stephan Hachinger (see arXiv:1201.1506) and is not distributed with TARDIS. TARDIS also requires a specific atomic datafile to use this module. This plasma option is included so that people who have access to and permission to use the necessary module may use it. Otherwise, the `recomb-NLTE` option provides a very accurate alternative approximation.
\ No newline at end of file
+Another `helium_treatment` option offered by TARDIS is `numerical-nlte`. The use of this plasma property requires an additional code that is the property of Stephan Hachinger (see arXiv:1201.1506) and is not distributed with TARDIS. TARDIS also requires a specific atomic datafile to use this module. This plasma option is included so that people who have access to and permission to use the necessary module may use it. Otherwise, the `recomb-NLTE` option provides a very accurate alternative approximation.
```

Comment:
What actually changed here? I can't tell from the changelog

Reply:
I did not edit or look at this section, so I'm not sure what, if anything, was changed here either.

Reply:
I have found that sometimes github just does that

Reply:
normally github does not change things. What does your local diff say?

Reply:
this is potentially a different newline character due to this being edited via Jupyter Notebook on a Windows OS.